test/test_bcm/test_bcm.py

```diff
@@ -0,0 +1,118 @@
+from ce_expansion.npdb import db_inter
+from pprint import pprint
+from ce_expansion.atomgraph.bcm import BCModel
+import numpy as np
+import os
+from pprint import pprint
+
+import ase.units as units
+
+this_dir = os.path.dirname(os.path.abspath(__file__))
+
+res = db_inter.get_bimet_result('AuCu', 'icosahedron', 55, n_metal1=22)
+
+atoms = res.atoms_obj
+bond_list = res.nanoparticle.bonds_list
+metal_types = ['Cu', 'Au', "Ag"]
+
+import ase.visualize
+#ase.visualize.view(atoms)
+
+# STEP 1
+# create a BCModel object
+
+
+#TODO: account for this case in bcm
+#metal_types = ['Cu', 'Au', 'au', 'ag']
+
+bcm = BCModel(atoms, bond_list, metal_types)
+assert bcm.metal_types == ['Ag', 'Au', 'Cu']
+
+print("STEP 1")
+
+# STEP 2
+# make sure everything passed in becomes an attribute
+print(bcm.atoms)
+print(bcm.bond_list.shape)
+print(bcm.metal_types)
+print("STEP 2")
+
+# STEP 2.5 DONE
+# this should print the total number of atoms
+print(len(bcm))
+print("STEP 2.5")
+
+# STEP 3
+# should calculate the following during initialization
+print("CE_BULK")
+pprint(bcm.ce_bulk)
+print("GAMMAS")
+pprint(bcm.gammas)
+print("STEP 3")
+
+# STEP 4
+# calculate precomputed factors array
+pprint(bcm.precomps)
+print("STEP 4")
+
+# STEP 5
+import os
+import ase.io as io
+
+ordering1 = np.zeros(len(bcm),int)
+ordering2 = np.ones(len(bcm),int)
+ordering3 = (atoms.symbols == metal_types[1]).astype(int)
```

Comment:
Very minor, but can you use `bcm.metal_types` rather than the defined variable? I only ask this for the sake of consistency when we convert this script to unit tests (we want to make sure we're testing the resolved metal_types attribute instead of the variable).